### DISCUSSION TITLE
如何启用文章图片点击放大的功能？

### DISCUSSION THREAD

**Discussion by sugarlesss** [HEIGE-PCloud/DoIt#142] [Q&A] [answered]
### Describe the feature you want 描述你的功能需求
当文章添加了一些分辨率比较高的图片时，因为无法点击放大图片，观看上会略有难受。如果能支持此功能，体验会好很多。

### Useful reference 有价值的参考

If available, provide useful links to fulfill the feature.
如果可以的话, 提供实现这个功能的相关参考链接.
1、一个点击图片放大显示的案例：https://blog.keyboardman.fun/share/1291.html

**Comment by sugarlesss** [comment]
类似于这篇文章中，像屏幕截图这种分辨率较高的图片，若有此功能的话，体验会好很多。
https://blog.keyboardman.fun/php/1053.html

**Comment by HEIGE-PCloud** [comment, marked as answer]
在 front matter 中开启 `lightgallery: true`，然后使用 image shortcode 引入图片来启用 lightgallery。(#118 )

相关文档：
1. https://hugodoit.pages.dev/zh-cn/theme-documentation-content/#front-matter
2. https://hugodoit.pages.dev/zh-cn/theme-documentation-extended-shortcodes/#image

相关讨论：
https://github.com/HEIGE-PCloud/DoIt/discussions/128

**Comment by sugarlesss** [comment]
非常感谢！
